src/java.desktop/share/classes/javax/swing/TablePrintable.java

```diff
@@ -395,11 +395,14 @@ public int print(Graphics graphics, PageFormat pageFormat, int pageIndex)
         // draw a box around the table
         g2d.setColor(Color.BLACK);
 
+        Rectangle bounds = table.getBounds();
+        bounds.x = bounds.y = 0;
+
```

Comment:
why was the x=0, y=0 necessary ? And if it wasn't (0,0) already that is a significant change

Reply:
It was (0,20) before which would make minRow as 1 instead of 0. If v see the computations in BasicTableUI it minRow will be 0 and maxRow will be max-1, hence in order the match both I had to make it (0,0), similar to BasicTableUI.